frameworks/helloworld/src/test/java/com/mesosphere/sdk/helloworld/scheduler/ServiceTest.java

```diff
@@ -270,7 +270,7 @@ public void testWorldDecommission() throws Exception {
 
         // Check initial plan state
         ticks.add(new ExpectDecommissionPlanProgress(Arrays.asList(
-                new StepCount("world-1", 6, 0, 0), new StepCount("world-0", 6, 0, 0))));
+                new StepCount("world-1", 9, 0, 0), new StepCount("world-0", 9, 0, 0))));
```

Comment:
In the long-term it would be nice to not have to manually manage these step counts.

Reply:
Switched to a variable that's assigned depending on whether default or custom executor is being exercised.